pl_bolts/datamodules/kitti_datamodule.py

```diff
@@ -1,12 +1,19 @@
+import importlib
 import os
 
 import torch
-import torchvision.transforms as transforms
 from pytorch_lightning import LightningDataModule
 from torch.utils.data import DataLoader
 from torch.utils.data.dataset import random_split
 
 from pl_bolts.datasets.kitti_dataset import KittiDataset
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+_TORCHVISION_AVAILABLE = importlib.util.find_spec("torchvision") is not None
```

Comment:
let's move these available packages to `pl_bolts/__init__`

Reply:
Shall we do this here or in #338?

Reply:
@Borda I'll quickly do this in #338 for all optional packages!